lib/msal-angular/README.md

```diff
@@ -44,8 +45,9 @@ At a minimum, `@azure/msal-angular` will follow the [support schedule of the mai
 
 | MSAL Angular version | MSAL support status     | Supported Angular versions |
 |----------------------|-------------------------|----------------------------|
-| MSAL Angular v2      | Active development      | 9, 10, 11, 12, 13, 14      |
-| MSAL Angular v1      | Active development      | 6, 7, 8, 9                 |
+| MSAL Angular v3      | Active development      | 15                         |
```

Comment:
So MSAL Angular v3 is only covering Angular 15?

Reply:
Yes. Due to breaking changes in Angular 13 and 14, MSAL Angular v3 being in Angular 15 is not backwards compatible. MSAL Angular v3 may cover Angular 16 once we test and determine compatibility.